src/test/java/com/znsio/sample/e2e/screen/web/AmazonCartWeb/ProductDetailsScreenWeb.java

```diff
@@ -1,2 +1,65 @@
-package com.znsio.sample.e2e.screen.web.AmazonCartWeb;public class ProductDetailsScreenWeb {
+package com.znsio.sample.e2e.screen.web.AmazonCartWeb;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.AmazonCart.ProductCartDetailsScreen;
+import com.znsio.sample.e2e.screen.AmazonCart.ProductDetailsScreen;
+import com.znsio.sample.e2e.screen.AmazonCart.SearchResultsScreen;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+
+import static com.znsio.e2e.tools.Wait.waitFor;
+
+public class ProductDetailsScreenWeb extends ProductDetailsScreen {
+
+    private static final String NOT_YET_IMPLEMENTED = "NOT_YET_IMPLEMENTED";
+    private static final Logger LOGGER = Logger.getLogger(ProductDetailsScreenWeb.class.getName());
+    private final Driver driver;
+    private final Visual visually;
+    private final String SCREEN_NAME = ProductDetailsScreenWeb.class.getSimpleName();
+    private final By ProductTitle = By.xpath("//span[@id='productTitle']");
```

Comment:
If we have id available, then why are we using xpath? You could have used By.id instead